### ISSUE TITLE
Trying to transfer ETH between wallets

### ISSUE THREAD

**Issue by GhostOutlaw** [ethereum/mist#3537]
- [Yes ] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

Version: `0.9.4`
OS & Version: windows
Node version: `geth 0.0.0` 
Number of blocks synchronized: 4,877,127
```



I am just trying to transfer ETH from my ETH wallet to exodus.
I've made 4 attempts now, with varying rates and amounts of gas and all 4 transactions are 
sitting at 0/12 confirmations. 
Some of these transfers are now 5 hours old.

What am I doing wrong?

http://prntscr.com/hxtn8q
http://prntscr.com/hxtned

**Comment by cac58** [comment]
I'm having the same issue

**Comment by Tho3as** [comment]
What gas price do you use?  The default is 1.  Due to the havy load on the Ethereum blockchain nowadays, you will have to increase your gas price.  I just did a transaction for 30 GWEI (21000 gas) and it was processed in less then 5 minutes.  You can use https://ethgasstation.info/ to determine your optimal gas price.

**Comment by GhostOutlaw** [comment]
I don't think there is a way to specify the gas price using this wallet.

**Comment by Tho3as** [comment]
With MyEtherWallet you can.  There is a slider at the top right of you screen.

**Comment by marcgarreau** [comment]
This may fall into a category of issues resolved with the [0.11.0 release](https://github.com/ethereum/mist/releases/tag/v0.11.0). Please give that a try and open if a new issue if appropriate.
